nautobot_ssot/integrations/infoblox/diffsync/models/infoblox.py

```diff
@@ -103,3 +103,9 @@ def update(self, attrs):
     #     """Delete an IP Address from Infoblox."""
     #     self.diffsync.conn.delete_host_record(self.get_identifiers()["address"])
     #     return super().delete()
+
+
+class InfobloxNamespace(Namespace):
```

Comment:
This should contain the CRUD methods and raise a NotImplementedError per our standard pattern.

Reply:
Do you mean like this?

```python
class InfobloxNamespace(Namespace):
    """Infoblox implementation of the Namespace model."""
    @classmethod
    def create(cls, diffsync, ids, attrs):
        """Don't allow creating Network Views in Infoblox"""
        diffsync.job.logger.error("Creating Network Views in Infoblox is not allowed.", extra={"object": ids["name"]})
        raise NotImplementedError
```

Reply:
Sure, that works!